scg-system/src/main/java/io/telicent/backup/services/DatasetBackupService.java

```diff
@@ -370,7 +373,11 @@ void executeRestoreTDB(DatasetGraph dsg, String tdbRestoreFile) throws IOExcepti
              InputStream gis = new GZIPInputStream(fis)) {
             Txn.executeWrite(dsg, () -> {
                 dsg.clear();
-                RDFDataMgr.read(dsg, gis, NQUADS);
+                RDFParser.create()
+                        .source(gis)
+                        .labelToNode(LabelToNodeGenerator.generate())
```

Comment:
Looks good

Do we also need to think about replacing the call to the generic Jena `Backup.backup()` at Line 201 with our own backup method that also ensures the Blank Node labels are consistent on backup?  Or does that code already just use the blank node labels as-is?